test/tests.js

```diff
@@ -142,6 +142,22 @@ module.exports.ok['zip64.zip'] = function(test) {
     });
 };
 
+module.exports.ok['pngAndTxt.zip'] = function(test) {
```

Comment:
Let's remove this file: we're not testing anything file-specific here.

Reply:
@antelle Done, I'm using another file for the test :ok_hand:

Reply:
@antelle BTW I didn't know whether you wanted me to delete the test or just replace the file, just in case, I'd like to keep the test to ensure the feature survives feature changes.